### DISCUSSION TITLE
[Feature] Prevent screen from turning off when app is open and backup is running

### DISCUSSION THREAD

**Discussion by klejejs** [immich-app/immich#3405] [Feature Request]
### The feature

Sometimes there are many photos that need to be uploaded and it takes a lot of time which means the screen automatically turns off. To prevent this, it has to be tapped regularly. It would be nice that if the foreground backup is running, the app prevented the screen from turning off, so the phone can be left somewhere to upload photos quicker than background backup. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by etnoy** [comment]
I second this opinion

**Comment by uhthomas** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/1629
